internal/command/format/diff_test.go

```diff
@@ -2859,6 +2867,7 @@ func TestResourceChange_nestedSet(t *testing.T) {
           - {
             - mount_point = "/var/diska" -> null
           },
+          # (1 unchanged element hidden)
```

Comment:
GitHub won't let me properly bracket the unchanged lines here to make this a real "suggestion" but here's a proposed alternative expected output for the indentation I was talking about in my review comment:

```
  # test_instance.example will be updated in-place
  ~ resource "test_instance" "example" {
      ~ ami   = "ami-BEFORE" -> "ami-AFTER"
      ~ disks = [
          + {
              + mount_point = "/var/diska"
              + size        = "50GB"
            },
          - {
              - mount_point = "/var/diska" -> null
            },
          # (1 unchanged element hidden)
        ]
        id    = "i-02ae66f368e8518a9"

      + root_block_device {
          + new_field   = "new_value"
          + volume_type = "gp2"
        }
      - root_block_device {
          - volume_type = "gp2" -> null
        }
    }
```

(That was kinda awkward to edit in the GitHub comment box so I probably didn't get it all exactly right, but the indentation of the `disks` contents is the main thing I'm getting at here.)